src/banking_stage.rs

```diff
@@ -79,7 +79,10 @@ impl BankingStage {
     ) -> Result<()> {
         let timer = Duration::new(1, 0);
         let recv_start = Instant::now();
-        let mms = verified_receiver.recv_timeout(timer)?;
+        let mut mms = verified_receiver.recv_timeout(timer)?;
+        while let Ok(mut nq) = verified_receiver.recv_timeout(Duration::from_millis(20)) {
```

Comment:
All others using this pattern do a `try_recv`? Why would we wait 20ms after we just timed out after 1000ms?

Reply:
I'll add documentation. The idea is that if multiple packets are received within a certain time (20ms) they are coalesced together. try_recv works only if the queue is backed up and there are no delay in subsequent packet receives.